src/components/BnCheckbox/BnCheckbox.vue

```diff
@@ -9,7 +9,8 @@ interface Props {
   value: valueTypes,
   name: string,
   color?: string,
-  rules?: RuleExpression<unknown>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  rules?: RuleExpression<any>,
```

Comment:
Here we could change this to what we discussed in PR [11](https://github.com/platanus/banano/pull/11)